src/components/SearchProducts/SearchProducts.tsx

```diff
@@ -0,0 +1,23 @@
+import { TouchableOpacity } from 'react-native';
+import { SearchProductCard } from './SearchProductCard';
+import { Product } from '../ProductSlide';
+
+interface Props {
+    product: Product,
+    position: number
+}
+
+const SearchProducts: React.FC<Props> = ({ product, position }) => {
+
+    const handleButtonClick = () => {
+        console.log(product.title)
+    }
+
+    return (
+        <TouchableOpacity onPress={() => handleButtonClick()}>
+            <SearchProductCard product={product} position={position}/>
+        </TouchableOpacity>
+    );
+}
+
```

Comment:
I think this is unnecessary, everything here is a `SearchProductCard`. Why not just pass `TouchableOpacity` and all the "click" functionality down the card? Do you agree?